### PR TITLE
Add Log Hierarchy, New Mod Logs Webhook

### PR DESCRIPTION
**Pull request by le-potate**
**Why this change was necessary**
Logs for the mods shouldn't be accessible in the log channel for dev

**What this change does**
-Instead of one Webhook there is now one Dev Webhook and one Mod Webhook, both optionally specified in `config.ini`.
-Info like who used `?mix` goes to the Mod Webhook (in cogs, use `self.bot.mod_logger.info("message")`). Logs useful for devs goes to the Dev Webhook (In cogs, use `self.bot.dev_logger.info("message")`
-All these logs still go in the log file

**More precisions**:
In `bot.py`:
-Add log hierarchy: Create new "sub"loggers `Canary.Dev` and `Canary.Mod`. The loglevel for `Canary.Dev` is the same than for `Canary`, that is the one specified in `config.ini`. The loglevel for `Canary.Mod` is info, since that is the level at which info for mods is logged.
-As it already was, the "main" `Canary` logger is handled by the file handler. (Everything logged to the "sub"loggers is also sent up to `Canary`, and thus to the log file.)
-The `_WebhookHandler` class is generalized to work with the two "sub"loggers.
-The two "sub"loggers are both handled by their respective webhook handler.
-In `config.ini` and `parser.ini`:
`LogWebhookID` and `LogWebhookToken` are replaced by `DevLogWebhookID`, `DevLogWebhookID`, `ModLogWebhookID` and `ModLogWebhookID`
In `memes.py`:
-`?mix` and `?bac` are now logged to the mod logger
In main.py:
Now sends the message that says output is logged to webhook if `dev_log_webhook_id` and `dev_log_webhook_token` have values
Other:
-Every mention of `bot(or self).logger` is changed to `bot(or self).dev_logger`. This is the logger that should now be used for logs that are useful for devs. Using `logger` only sends the log to the log file, not the dev webhook. There were mentions in `bot.py` and `main.py`
-Updated the README

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -62,14 +62,15 @@
...
-    if bot.config.log_webhook_id and bot.config.log_webhook_token:
+    if bot.config.dev_log_webhook_id and bot.config.dev_log_webhook_token:
         webhook_string = " and to the log webhook"
     else:
         webhook_string = ""
     sys.stdout.write(f'Bot is ready, program output will be written to a '
                      f'log file{webhook_string}.\n')
     sys.stdout.flush()
-    bot.logger.info('Logged in as {} ({})'.format(bot.user.name, bot.user.id))
+    bot.dev_logger.info('Logged in as {} ({})'.format(bot.user.name,
```
Comment (davidlougheed) [inline Main.py:72]: we can use f-strings now, thanks yapf!!!!! :)))))))))))

File: Main.py [hunk 2]
```diff
@@ -164,15 +165,16 @@ async def backup(ctx):
...
 if __name__ == "__main__":
     for extension in startup:
         try:
             bot.load_extension(extension)
         except Exception as e:
-            bot.logger.warning('Failed to load extension {}\n{}: {}'.format(
-                extension,
-                type(e).__name__, e))
+            bot.dev_logger.warning(
+                'Failed to load extension {}\n{}: {}'.format(
```
Comment (davidlougheed) [inline Main.py:177]: f-string

**Review by davidlougheed — APPROVED** [review]

File: Main.py (hunk 1 above)
Comment (le-potate) [reply Main.py]: Resolved in 380013c

File: Main.py (hunk 2 above)
Comment (le-potate) [reply Main.py]: Resolved in 380013c

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**